packages/core/src/shared/utilities/workspaceUtils.ts

```diff
@@ -268,30 +269,44 @@ export function checkUnsavedChanges(): boolean {
     return vscode.workspace.textDocuments.some((doc) => doc.isDirty)
 }
 
+export const DefaultExtraExcludePatterns = [
```

Comment:
codebase conventions
```suggestion
export const defaultExtraExcludePatterns = [
```